straxbra/plugins.py

```diff
@@ -372,28 +376,50 @@ def compute(self, peaks):
                  default=100)
 )
 class PeakPositions(strax.Plugin):
-    """
-    I actually wrote this
+    '''
+    Position Reconstruction for XeBRA
+    
     Version 0.0.1: weighted sum
-    Version 0.0.2: hopefully LRF or better
-    """
-    __version__ = "0.0.1"
+    Version 0.0.2: LRF
+    
+    Status: July 2019, Version 0.0.2
+    
+    Position reconstruction for XeBRA following the position reconstruction algorithm of Mercury 
+    (employed in the LUX experiment, originally developed for the ZEPLIN-III dark matter experiment) 
+    with light response functions (see https://arxiv.org/abs/1710.02752v2 and 
+    https://arxiv.org/abs/1112.1481 ).
+    '''
+    __version__ = "0.0.2"
     dtype = [('x', np.float32,
-              'Reconstructed S2 X position (cm), uncorrected'),
+              'Reconstructed S2 X position (mm), uncorrected'),
              ('y', np.float32,
-              'Reconstructed S2 Y position (cm), uncorrected')]
+              'Reconstructed S2 Y position (mm), uncorrected')]
     depends_on = ('peaks',)
 
     parallel = False
 
     def setup(self):
-
+        # PMT mask
         self.pmt_mask = np.zeros_like(self.config['to_pe'], dtype=np.bool)
         self.pmt_mask[self.config['top_pmts']] = self.config['to_pe'][self.config['top_pmts']] > 0
```

Comment:
Do we want to assume strictly non-negative PMT gain, or should we add a `min_gain` option like above?